### ISSUE TITLE
Draeni Racials Dismissable (BB #711)

### ISSUE THREAD

**Issue by OregonAsari** [OregonCore/OregonCore#711]
This issue was migrated from bitbucket.
**Original Reporter:** LordUsagi
**Original Date:** 09.10.2014 14:29:34 GMT+0000
**Original Priority:** trivial
**Original Type:** bug
**Original State:** closed
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/711

<hr>

Related issue #710

Currently you can right click dismiss Draeni Passive racials. Probably missing a check for spell attributes, or missing an attribute in general.

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 11.10.2014 16:23:15 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/711/#comment-12909127

<hr>

Something to take note of, the aura isn't removable by other players, but simply the initial draeni. It's not possible to recast this spell either.

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Flywithme
**Original Date:** 12.10.2014 10:38:35 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/711/#comment-12915594

<hr>

May be necessary to add an attribute: 

```
#!c++

SPELL_ATTR_CANT_CANCEL                    0x80000000
```

?

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 13.10.2014 21:21:12 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/711/#comment-12944557

<hr>

Most likely.

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 05.11.2014 01:09:49 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/711/#comment-13425212

<hr>

[Core/Spells] Implement IsPassive and prevent dismissing passive auras.
              Closes #711.

→ <<cset 0b0d07201d9f>>
